src/commands/initProjectForVSCode/InitVSCodeStep/DotnetInitVSCodeStep.ts

```diff
@@ -63,11 +63,11 @@ export class DotnetInitVSCodeStep extends InitVSCodeStepBase {
 
         const projFilePath: string = path.join(projectPath, projFileName);
 
-        const versionInProjFile: string = await dotnetUtils.getPropertyInProjFile(projFilePath, 'AzureFunctionsVersion');
+        const versionInProjFile: FuncVersion | undefined = await dotnetUtils.getFuncVersion(projFilePath);
```

Comment:
I want this to be the exact version in the proj file before `tryParseFuncVersion` is called. We collect this in telemetry in the next line, and it's helpful to know what "unexpected" versions people are using

Reply:
reverted the change